Release note .NET provider 8.4.0 (#1098)
Azure/AppConfiguration: releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

# Microsoft.Extensions.Configuration.AzureAppConfiguration
[Source code][source_code] | [Package (NuGet)][package]

## 8.4.0 - September 19th, 2025
### Enhancements

> * Added support for parsing json values with comments. [#685](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/685)

Reviewer: ```suggestion
* Added support for accepting JSON values with comments. [#685](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/685)
```

> * Added support for snapshot references. If a configuration setting's value is a snapshot reference, loading it automatically pulls all key-values and feature flags from the referenced snapshot into the configuration. This enables dynamically switching between snapshots at runtime. [#689](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/689)

Reviewer 2: ```suggestion
* Added support for snapshot references. If a configuration setting's value is a snapshot reference, loading it automatically loads all key-values from the referenced snapshot. This enables dynamically switching between snapshots at runtime. [#689](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/689)
```

> * Updated dependency package versions. [#695](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/695)

Author: I refered to [7.3.0](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md#730---july-16th-2024)

But I prefer to omit it. Remove this from the current release note.
